cloudformation/bulkExport.yaml

```diff
@@ -6,6 +6,11 @@
 Resources:
   GlueJobRelatedLambdaRole:
     Type: AWS::IAM::Role
+    Metadata:
+      cfn_nag:
+        rules_to_suppress:
+          - id: W11
+            reason: '* only the the glue jobs and is required due to glue requirements'

```

Comment:
This reason is confusing. Can we rephrase it?

Reply:
updated